auth/server/src/main/java/com/seagates3/dao/ldap/FileStore.java

```diff
@@ -54,11 +55,12 @@
       oos.close();
     }
     catch (IOException e) {
-      LOGGER.error("Exception occurred while saving policy into file - " + e);
+      LOGGER.error("Exception occurred while saving data into file - " + e);
+      throw new DataAccessException("Exception occurred while saving file");
     }
   }
 
-  @Override public Object find(String key, Object obj,
+  @Override public Object find(String key, Object obj, Object obj2,
                                String prefix) throws DataAccessException {
     return savedDataMap.get(key);
```

Comment:
obj and obj2 are not used, are they required here

Reply:
No. Not required but method prototype made generic for all AUthStorages in AUthStore class